src/main/java/ama/post/Post.java

```diff
@@ -36,5 +37,19 @@ public void setText(String text) {
         this.text = text;
     }
 
+    public User getUser() {return op;}
+
+    public void setUser(User user) {
+        this.op = user;
+    }
+
     public Long getId() { return id; }
+
+    public Collection<CommentPost> getReplies() {
+        return replies;
+    }
+
+    public void setReplies(Collection<CommentPost> replies) {
```

Comment:
This never actually gets used for adding to the replies collection from what I can see. Is that taken care of by @OneToMany(mappedBy="context")?